public/js/office.web.js

```diff
@@ -23,83 +20,83 @@ $(() => {
       auth2.signOut().then(() => {
         matrixProfile.terminate();
         auth2.disconnect();
-        window.location = '/';
+
+        redirectToHome();
       });
     });
   }
 
-
   function removeUser(userId) {
     $(`#${userId}`).remove();
   }
 
   function showUserInRoom(user, room) {
+    let userView = $(`#${user.id}`);
 
-    var userView = $(`#${user.id}`).length;
-    if (userView == 0) {
+    if (!userView.length) {
       userView = $(`<div  id="${user.id}" class="thumbnail user-room"><img user-presence class="rounded-circle" style="margin:2px;display:flex;" user-id="${user.id}" title="${user.name}" width="50px" src="${user.imageUrl}"></div>`);
-     } else {
-       userView = $(`#${user.id}`).detach();
+    } else {
+      userView.detach();
     }
 
     userInRoomDecorator(user, room);
-    userInMeetDecorator(user,userView);
+    userInMeetDecorator(user, userView);
 
     $(`#${room}`).append(userView);
-
-    
   }
 
-  function initGetUserMenu(officeEvents){
-      $("[user-presence]").initialize(function(){
-        $(this).contextMenu({
-            menuSelector: "#getUserMenu",
-            menuSelected: function (invokedOn, selectedMenu) {
-            var userId = $(invokedOn).attr("user-id");
-            var roomId = getLastRoom(matrixProfile);  
-            officeEvents.callUserForMyRoom(userId,roomId);
-          }
-        });
+  function initGetUserMenu(officeEvents) {
+    $("[user-presence]").initialize(function() {
+      $(this).contextMenu({
+        menuSelector: "#getUserMenu",
+        menuSelected: function (invokedOn, selectedMenu) {
+          const userId = $(invokedOn).attr("user-id");
+          const roomId = getLastRoom(matrixProfile);
+          officeEvents.callUserForMyRoom(userId,roomId);
+        }
       });
+    });
   }
 
-  function userInMeetDecorator(user,userView){
-
-    var userMeetClass = "rounded-circle user-not-in-call user-room"
+  function userInMeetDecorator(user, userView){
+    const userMeetDefaultClasses = "rounded-circle user-room";
 
-    if(user.inMeet!=undefined && user.inMeet){
-      userMeetClass = "rounded-circle user-in-call user-room";
+    if (user.inMeet) {
```

Comment:
This check is unnecessary (and introduce a bug), given that toggleClass will handle with classes when user enter or leave a meet.